tests/functional/firefox/new/test_download.py

```diff
@@ -38,7 +35,7 @@ def test_other_platforms_modal(base_url, selenium):
 @pytest.mark.nondestructive
 @pytest.mark.skip_if_not_firefox(reason='Join Firefox form is only displayed to Firefox users')
 def test_firefox_account_modal(base_url, selenium):
```

Comment:
Same for this test, as we no longer promote anything to existing Firefox users trying to download?